src/Codeception/Lib/Interfaces/CustomCommand.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Codeception\Lib\Interfaces;
```

Comment:
I'd like to  move that to Codeception\CustomCommandInterface to make it simpler to use. 
Also `Lib\Interface` is more like internal stuff, and actually interfaces there are related to modules